pkg/sensors/tracing/lists_windows.go

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package tracing
+
+import (
+	"fmt"
+
+	"github.com/cilium/tetragon/pkg/k8s/apis/cilium.io/v1alpha1"
+)
+
+// isList checks if a value specifies a list, and if so it returns it (or nil if list does not exist)
+func isList(val string, lists []v1alpha1.ListSpec) (bool, *v1alpha1.ListSpec) {
+	return false, nil
+}
+
+func preValidateLists(lists []v1alpha1.ListSpec) (err error) {
+	return fmt.Errorf("not supported on windows")
```

Comment:
Because, there are no arguments, you could use errors.New() here.
But, I think it would even be better to have a global variable and use that everywhere. I.e.,

```
var (
    notSupporterErr = errors.New("not supported on windows")
)
```